_includes/menu-item-ids/core-menu-items.md

```diff
@@ -12,6 +12,7 @@
 | `fontformats`        | Format                                                                                                   | Dropdown list with font families to apply to selection.                        |
 | `fontsizes`          | Format                                                                                                   | Dropdown list with font sizes to apply to selection.                           |
 | `italic`             | Format                                                                                                   | Applies italic format to current selection.                                    |
+| `language`           | Format                                                                                                   | Dropdown list with languages to apply to selection. This item requires the [`content_langs` option]({{site.baseurl}}/configure/content-appearance/#content_langs).<br />{{site.requires_5_9v}}     |
```

Comment:
```suggestion
| `language`           | Format                                                                                                   | Dropdown list with languages to apply to the selection. This item requires the [`content_langs` option]({{site.baseurl}}/configure/content-appearance/#content_langs).<br />{{site.requires_5_9v}}     |
```

Reply:
I had this, and James pointed out that the other rows in this table all avoid `the` for some reason

Reply:
This URL is also now wrong, since the setting was moved to localization.

With regards to the original point, all the existing ones are also broken, but that doesn't mean we should be adding more broken content.